src/js/admin/index.js

```diff
@@ -0,0 +1,20 @@
+document.querySelector( '.media-single-image button.browse' ).addEventListener( 'click', selectImage );
+
+function selectImage() {
+	const optionName = this.dataset.option;
+
+	const imageFrame = wp.media( {
+		multiple: false,
+		library: {
+			type: 'image',
+		},
+	} );
+
+	imageFrame.on( 'select', function() {
+		const url = imageFrame.state().get( 'selection' ).first().toJSON().url;
+		const inputSelector = '#media-single-image-' + optionName + ' input.file-path';
+		document.querySelector( inputSelector ).value = url;
```

Comment:
Something similar here. What if we can't select the DOM element?

Reply:
- Displaying alerts with messages would imply JS localization. Do you want us to go that route?
- I also think the above aren't dealbreakers. If the browser can't use the media library correctly as well as detect the input, the user already has much bigger problems. Are you confirming we should get into that level of checking?

Reply:
You have a point. The properties come from a fairly stable object, so let's go with your solution. Resolving this conversation for now.